<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FTPClientClass</class>
 <widget class="QMainWindow" name="FTPClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quick FTP</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButtonUpload">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>上传</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonRenameLocal">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>重命名</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonDeleteLocal">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButtonParentDirLocal">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditPathLocal"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="listWidgetFileLocal"/>
        </item>
        <item>
         <widget class="QLabel" name="labelSelectedFileLocal">
          <property name="text">
           <string>已选:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="pushButtonDownload">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>下载</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonRenameRemote">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>重命名</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonDeleteRemote">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButtonParentDirRemote">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditPathRemote">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="listWidgetFileRemote"/>
        </item>
        <item>
         <widget class="QLabel" name="labelSelectedFileRemote">
          <property name="text">
           <string>已选:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1032</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSession">
    <property name="title">
     <string>会话(&amp;E)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>工具(&amp;T)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
   </widget>
   <addaction name="menuSession"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionxinjian">
   <property name="text">
    <string>xinjian</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>连接</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>断开</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="text">
    <string>日志</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FTPClient.qrc"/>
 </resources>
 <connections/>
</ui>
